sdk/search/azure-search-documents/src/main/java/module-info.java

```diff
@@ -5,6 +5,7 @@
     requires transitive com.azure.core;
     requires com.fasterxml.jackson.datatype.jsr310;
     requires jakarta.activation;
+    requires transitive com.azure.core.serializer.json.jackson;
```

Comment:
This should only `require`. `transitive` is when it is used as part of the public API.